### PR TITLE
Release notes for MTA 5.1.1

### PR DESCRIPTION
**Pull request by RichardHoch**
Resolves https://issues.redhat.com/browse/WINDUP-2944
Preview: http://file.emea.redhat.com/rhoch/18012021/rn_511/html-single/

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick : please review.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 1]
```diff
@@ -5,14 +5,26 @@
...
+|MTA reports load slowly, do not include Target Runtime labels on Application list, and do not have working feedback controls.
+
+|link:https://issues.redhat.com/browse/WINDUP-2333[WINDUP-2333]
+|The MTA web console installed on OpenShift by means of a template cannot access the reports using HTTPS.
+
+|link:https://issues.redhat.com/browse/WINDUP-2801[WINDUP-2801]
+|The MTA web console installed on OpenShift by means of the MTA Operator cannot access the reports using HTTPS.
+
+|link:https://issues.redhat.com/browse/WINDUP-2910[WINDUP-2910]
+|"Unparseable Class File" errors are generated when MTA analyzes applications built with JDK 11.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:26]: Passive voice. Change to active.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 2]
```diff
@@ -5,14 +5,26 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12356591[{ProductVersion} resolved issues] in Jira.
+For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{ProductVersion} resolved issues] in Jira.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:8]: I see a problem here. Phil named the filter "5.1.1 Resolved issues" but it is 5.1.0.Final and 5.1.1. Ask him to change this to "5.1 resolved issues" ASAP. Change the link text from `{ProductVersion} resolved issues` to `{DocInfoProductNumber} resolved issues`.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 3]
```diff
@@ -5,14 +5,26 @@
...
 [cols="25%,75%",options="header"]
 |====
 |ID
 |Summary
 
+|link:https://issues.redhat.com/browse/WINDUP-2941[WINDUP-2941]
+|MTA reports load slowly, do not include Target Runtime labels on Application list, and do not have working feedback controls.
+
+|link:https://issues.redhat.com/browse/WINDUP-2333[WINDUP-2333]
+|The MTA web console installed on OpenShift by means of a template cannot access the reports using HTTPS.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:20]: This sounds like the reports are using HTTPS, which is not the actual meaning.

**Comment by RichardHoch** [comment]
@apinnick: please review -- the preview has been updated.

**Pull request merged**